### DISCUSSION TITLE
Remove documentation about Microsoft Extensions

### DISCUSSION THREAD

**Discussion by schalkneethling** [mdn/mdn-community#222] [Content] [answered]
Following on from this [draft pull request](https://github.com/mdn/content/pull/18750), there is a clear need for a discussion so we can conclude how to move forward. To add some context to the discussion:

> This PR removes all of the documentation regarding Microsoft proprietary events, properties, etc. from MDN Web Docs.  They can be removed per BCD's irrelevance guidelines, since they have been removed from Edge over two years ago, and have not been included in a browser version that has been released within the last five years.  Related BCD PR: https://github.com/mdn/browser-compat-data/pull/17120

There are several issues raised with the pull request in its current state:

- https://github.com/mdn/content/pull/18750#pullrequestreview-1049811624
- https://github.com/mdn/content/pull/18750#issuecomment-1234410330
- https://github.com/mdn/content/pull/18750#pullrequestreview-1094096576

Let’s have a discussion about this and get to a conclusion that would be a good solution for our readers.

**Comment by bsmth** [comment, marked as answer]
This was completed in the following PRs:

- [x] https://github.com/mdn/content/pull/24790
- [x] https://github.com/mdn/content/pull/24793
- [x] https://github.com/mdn/content/pull/24794
